src/main/java/serverutils/handlers/ServerUtilitiesServerEventHandler.java

```diff
@@ -1,5 +1,7 @@
 package serverutils.handlers;
 
+import static net.minecraft.realms.RealmsMth.floor;
```

Comment:
That is a client-side only class and is definitely not the floor you want.

Reply:
oh maybe i add the wrong class. What class should i import instead ?

Reply:
I have no clue which `floor` the author intended to use but
`MathHelper.floor_double()` from Minecraft or the native Java `Math.floor()` both work. The java one will need a cast to int though.

Reply:
@Novie53 can you correct the import. Thanks